app/services/foreman/ipxe/message_renderer.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Foreman
+  module Ipxe
+    class MessageRenderer
+      attr_accessor :message
+
+      def initialize(message:)
+        @message = message
+      end
+
+      def to_s
+        <<~MESSAGE
+        #!ipxe
+        echo #{message}
+        shell
+        MESSAGE
+      end
+    end
+  end
+end
```

Comment:
This is nice indeed, message on screen is great. But seriously, 21 lines and a class to generate three lines? Why not to do it in the `unattended_controller.rb`? Ignore the metric if there's complain, I do not take them seriously.

Reply:
Oh after more lines I see the `StubRenderer` this is fine then.